src/odom_to_tf.cpp

```diff
@@ -0,0 +1,67 @@
+#include <functional>
+#include <memory>
+#include <string>
+
+#include "rclcpp/rclcpp.hpp"
+#include "nav_msgs/msg/odometry.hpp"
+#include "tf2_msgs/msg/tf_message.hpp"
+#include "tf2_ros/qos.hpp"
+
+using std::placeholders::_1;
+
+class OdomToTfNode : public rclcpp::Node
+{
+  public:
+    OdomToTfNode()
+    : Node("odom_to_tf")
+    {
+      this->declare_parameter<std::string>("gazebo_entity", "marta");
+      this->declare_parameter<std::string>("body_tf_name", "odom");
+
+      std::string gazebo_entity;
+
+      this->get_parameter("gazebo_entity", gazebo_entity);
+      this->get_parameter("body_tf_name", body_tf_name_);
+
```

Comment:
```suggestion
```suggestion

      // The subscription requires sensor data quality of service. Otherwise the messages are not received.
```
```

Reply:
Important comment, thx!